src/Storage/Repository/PgInstanceRepository.cs

```diff
@@ -362,6 +362,9 @@ private async Task<InstanceQueryResponse> GetInstancesInternal(
                 ToExternal(instance);
             }
 
+            // Present instance data elements in chronological order
+            instance.Data = [.. instanceData.OrderBy(x => x.Created)];
```

Comment:
I think I would prefer old school ToList here:
```suggestion
            instance.Data = instanceData.OrderBy(x => x.Created).ToList();
```

Reply:
Move the ToExternal statement to below this. It doesn't have a practical effect, but I want it to be done last.